packages/mint-components/src/components/sqm-rewards-table/sqm-rewards-table.tsx

```diff
@@ -206,6 +207,7 @@ function useRewardsTableDemo(
     const rows =
       cellsPromise && (await Promise.all(cellsPromise)).filter((i) => i);
 
+    // @ts-ignore
```

Comment:
Flagging ignore

Reply:
👍 i think a typescript updated has caused this to fail when it didn't previously, not going to worry about it at the moment 😢